eb-cli/src/main/java/io/engineblock/cli/EBCLIScriptAssembly.java

```diff
@@ -41,7 +42,22 @@ public static ScriptData assembleScript(EBCLIOptions options) {
                 case start: // start activity
                 case run: // run activity
                     // Sanity check that this can parse before using it
-                    ActivityDef activityDef = ActivityDef.parseActivityDef(cmdSpec);
+                    activityDef = ActivityDef.parseActivityDef(cmdSpec);
+                    sb.append("// from CLI as ").append(cmd).append("\n")
+                            .append("scenario.").append(cmdType.toString()).append("(\"")
+                            .append(cmdSpec)
+                            .append("\");\n");
+                    // workaroud for perf issue
+                    sb.append("// from CLI as ").append(cmd).append("\n");
+                    sb.append("activities.test.noop=\"noop\"");
+                    if (!cmdSpec.endsWith("\n")) {
+                        sb.append("\n");
+                    }
+                    break;
+                case start2: // run activity
+                case run2: // run activity
+                    // Sanity check that this can parse before using it
+                    activityDef = ActivityDef.parseActivityDef(cmdSpec);
                     sb.append("// from CLI as ").append(cmd).append("\n")
                             .append("scenario.").append(cmdType.toString()).append("(\"")
```

Comment:
I believe this will result in script like 'scenario.run2...' which must be a supported method on ScenarioController, as added to the scripting environment in https://github.com/engineblock/engineblock/blob/b7f18a531d769f39d5f14632850af92362b29ba1/eb-core/src/main/java/io/engineblock/script/Scenario.java#L104

Thus, you may need to alias the cmd or cmd type back to a form that doesn't include the '2'

Reply:
good catch, addressed in next commit